### PR TITLE
db: only select repo.name when that’s all we need

### PR DESCRIPTION
**Pull request by unknwon**
Follow up of #16006

### CODE REVIEW

File: internal/db/repos.go [hunk 1]
```diff
@@ -1014,28 +1014,25 @@ func (*RepoStore) listSQL(opt ReposListOptions) (conds []*sqlf.Query, err error)
...
-
 	authzConds, err := authzQueryConds(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	// TODO(jchen): Only select name
 	const fmtString = `
-SELECT %s FROM repo
+SELECT repo.name FROM repo
```
Comment (ryanslade) [inline internal/db/repos.go:1023]: Until we add a unique constraint (which I've just PR'd) we should select distinct names here

Comment (ryanslade) [reply internal/db/repos.go:1023]: Actually, it should always be distinct since more than one user could add the same repo

**Review by ryanslade — APPROVED** [review]
Approved to unblock

File: internal/db/repos.go (hunk 1 above)
Comment (unknwon) [reply internal/db/repos.go]: Gotcha, updated!

**Pull request merged**